docs/model_zoo.md

```diff
@@ -8,7 +8,7 @@ To propose a model for inclusion, please submit a [pull request](https://github.
 
 | Model File | Type | Dataset | Source | Size | Download |
 | --- | --- | --- | --- | --- | --- |
-| [AlexNet](#alexnet) | Image Classification | ImageNet | ONNX | 233 MB | [.mar](https://s3.amazonaws.com/model-server/model_archive_1.0/onnx-alexnet/alexnet.mar) |
+| [AlexNet](#alexnet) | Image Classification | ImageNet | ONNX | 233 MB | [.mar](https://s3.amazonaws.com/model-server/models/alexnet/alexnet.mar) |
```

Comment:
We should put all 1.0 model archive file in the same location on S3.

Reply:
I put it where our other models are. There are other models in this bucket as well.

Reply:
OH, I think you created a new bucket for 1.0 archive. Not sure why you created a new bucket, they are all just models, we could have let them all live in the same bucket. But, anyways, I changed the location.